### ISSUE TITLE
Typo: Type changes in typescript/hit-counter

### ISSUE THREAD

**Issue by rmalecky** [aws-samples/aws-cdk-intro-workshop#120]
### What is the problem?
In the Typescript/Hit Counter the type of `downstream` is `lambda.IFunction` at the start of the exercise and `lambda.Function` later.

20-typescript/40-hit-counter/100-api.html
```ts
export interface HitCounterProps {
  /** the function for which we want to count url hits **/
  downstream: lambda.IFunction;
}
```

in 20-typescript/40-hit-counter/300-resources.html
```ts
export interface HitCounterProps {
  /** the function for which we want to count url hits **/
  downstream: lambda.Function;
}
```

I'm new to typescript, sorry if this isn't a typo.

**Comment by NGL321** [comment]
@rmalecky Good catch! Looks like a typo to me (it should be the interface IFunction rather than the class Function both times).

Feel free to put in a PR to fix, otherwise I will fix it soon!

😸

**Comment by NGL321** [comment]
Fixed by #130. Just missed the close
